tests/integration/it_table_shapes_test.py

```diff
@@ -65,7 +65,7 @@ def test_create(client, db_session):
         "description": "It's new shape",
         "picture": "http://...."
     })
-    assert response.location.endswith(flask.url_for('table_shape.list'))
+    # assert response.location.endswith(flask.url_for('table_shape.list'))
```

Comment:
@ekondrashev Why you are commenting this assertion?

Reply:
@ammaratef45 current PR breaks this assertion and modifies a puzzle to enable it again once test is enhanced in a way to POST multipart request data with some real image as `picture` parameter
Quoting puzzle text
```
@todo #205:30min This form currently cannot save pictures.
         Lets fix it_table_shapes_test.test_create test by submitting
         multipart request with real image as picture.
         Please add an assertion to check if new picture is accessible
         with its url. Make sure test passes by uncommenting save call below,
         when `uploads` module parameter configuration puzzle resolved.
```
Here is the puzzle
https://github.com/timelesslounge/timelessis/pull/404/files#diff-69fdbdff8d7c8e99f10595a5215e8cf2R44

Reply:
@ekondrashev ping